### ISSUE TITLE
bot errors when attempting to automatically trigger lite build

### ISSUE THREAD

**Issue by richardlau** [nodejs/github-bot#229]
It looks like recent PR's raised by collaborators (e.g. https://github.com/nodejs/node/pull/26992#issuecomment-478070070) are getting this comment:

> Sadly, an error occurred when I tried to trigger a build. :(

**Comment by phillipj** [comment]
Found this in the logs:

```
16:49:15.309Z ERROR bot: Error while triggering Jenkins build (req_id=955fe640-5242-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26992, action=pull_request.opened)
    Error: Expected 200 from Jenkins, got 400
        at Request.request.post [as _callback] (/.../github-bot/scripts/trigger-jenkins-build.js:73:17)
```

Haven't dug into the details yet, but seeing that same type of error has increased a lot since yesterday, I'm starting to suspect it's due to some recent changes in the bot code or Jenkins.

```
03:04:19.071Z ERROR bot: Error while triggering Jenkins build (req_id=57a65730-51cf-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26974, action=pull_request.opened)
03:20:18.277Z ERROR bot: Error while triggering Jenkins build (req_id=933cf090-51d1-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26976, action=pull_request.opened)
10:28:16.390Z ERROR bot: Error while triggering Jenkins build (req_id=5c91eaf0-520d-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26978, action=pull_request.opened)
13:19:47.621Z ERROR bot: Error while triggering Jenkins build (req_id=5297abd0-5225-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26980, action=pull_request.opened)
14:40:35.765Z ERROR bot: Error while triggering Jenkins build (req_id=9c4ef980-5230-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26982, action=pull_request.opened)
14:54:30.862Z ERROR bot: Error while triggering Jenkins build (req_id=8e513580-5232-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26983, action=pull_request.opened)
14:55:54.661Z ERROR bot: Error while triggering Jenkins build (req_id=c05773f0-5232-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26984, action=pull_request.opened)
15:43:53.032Z ERROR bot: Error while triggering Jenkins build (req_id=73b424b0-5239-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26986, action=pull_request.opened)
16:04:09.647Z ERROR bot: Error while triggering Jenkins build (req_id=490af100-523c-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26988, action=pull_request.opened)
16:17:04.874Z ERROR bot: Error while triggering Jenkins build (req_id=170f4f00-523e-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26989, action=pull_request.opened)
16:49:15.309Z ERROR bot: Error while triggering Jenkins build (req_id=955fe640-5242-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26992, action=pull_request.opened)
17:17:50.481Z ERROR bot: Error while triggering Jenkins build (req_id=941eb6e0-5246-11e9-a673-1d9c0b118da9, pr=nodejs/node/#26994, action=pull_request.opened)
```

**Comment by refack** [comment]
Ok. Been thinking hard what did I do that could break this... I thought to myself "nothing", but then I figured it out... I reduced the number of required parameters to trigger `node-test-pull-request-lite-pipeline`.

tl;dr is working again:
![image](https://user-images.githubusercontent.com/96947/55264150-d0c25e00-5249-11e9-9486-d8a604841220.png)

**Comment by phillipj** [comment]
Ooooh thanks a bunch!
